### PR TITLE
Simplify example solution

### PR DESCRIPTION
**Pull request by Marcos-cat**

### CODE REVIEW

**Comment by Marcos-cat** [comment]
I'm also unsure of the use of `comptime`, as constant bindings are only evaluated once to begin with, so it seems redundant

**Comment by ErikSchierboom** [comment]
> I'm also unsure of the use of comptime, as constant bindings are only evaluated once to begin with, so it seems redundant

Ah, I didn't know that.

File: exercises/practice/roman-numerals/.meta/example.ua [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Hundreds      ← {"" "C" "CC" "CCC" "CD" "D" "DC" "DCC" "DCCC" "CM"}
 Tens          ← {"" "X" "XX" "XXX" "XL" "L" "LX" "LXX" "LXXX" "XC"}
 Ones          ← {"" "I" "II" "III" "IV" "V" "VI" "VII" "VIII" "IX"}
-Lookup        ← comptime(☇ 0 ≡≡(⍚/◇⊂) ⊞(⊂⊂) Hundreds Tens Ones)
-Thousands     ← |1 ↯: "M" ⊂[] ⌊÷1000
+Lookup        ← comptime(♭ ⊞⍚(⊂⊂) Hundreds Tens Ones)
```
Comment (ErikSchierboom) [inline exercises/practice/roman-numerals/.meta/example.ua:4]:
```suggestion
Lookup        ← ♭ ⊞⍚(⊂⊂) Hundreds Tens Ones
```

**Review by ErikSchierboom — APPROVED** [review]
Lovely improvements!

**Pull request merged**